backend/package.json

```diff
@@ -5,7 +5,7 @@
   "type": "module",
   "main": "index.js",
   "scripts": {
-    "dev": "npm install && tsx watch src/index.ts",
+    "dev": "npm install && tsc --watch",
     "test": "NODE_ENV=test jest --coverage --detectOpenHandles",
```

Comment:
Awesome! Will remove. Hopefully that fixes your issue with running the command.

As a side note, if you run `npm run test` all the old tests will fail (or most will). You probably already noticed that so be sure to isolate only the tests we're working on to prevent any unforeseen issues.